src/router/routes.ts

```diff
@@ -44,8 +44,38 @@ export const routes = [
     }
   },
   {
-    path: '/restoration',
-    name: RouteNames.RESTORATION,
+    path: '/limitedRestorationToFull',
+    name: RouteNames.RESTORATION_CONVERSION,
+    component: Restoration,
+    meta: {
+      requiresAuth: true,
+      isStaffOnly: true,
+      filingType: FilingTypes.RESTORATION
+    }
+  },
+  {
+    path: '/limitedRestorationExtension',
+    name: RouteNames.RESTORATION_EXTENSION,
+    component: Restoration,
+    meta: {
+      requiresAuth: true,
+      isStaffOnly: true,
+      filingType: FilingTypes.RESTORATION
+    }
+  },
+  {
+    path: '/fullRestoration',
```

Comment:
What's the plan here, since Edit UI doesn't process full or limited resto filings?

Reply:
That's my mistake.  I'll remove those routes.